app.js

```diff
@@ -24,4 +24,6 @@ app.use("/", indexRouter);
 app.use("/users", usersRouter);
 app.use("/courses", coursesRouter);
 
+let server = app.listen(3000, () => console.log('Listening on port: ', server.address().port))
```

Comment:
In fact, there's another file which already start the app, it's bin/www. There's no need to have a app.listen here

Reply:
That file is launched with npm start

Reply:
Thanks